docs/deployment/customization/application.properties-Reference.md

```diff
@@ -22,14 +22,8 @@ For example:
 jdbc:mysql://localhost:3306/cbiodb?zeroDateTimeBehavior=convertToNull&useSSL=false
 ```
 
-:warning: The fields `db.host` and `db.portal_db_name` and `db.use_ssl` are deprecated. It is required to configure the database connection using
-the `db.connection_string` instead.
-
-`db.tomcat_resource_name` is required in order to work with the tomcat database connection pool and should have the default value jdbc/cbioportal in order to work correctly with the your WAR file.
-
-```
-db.tomcat_resource_name=jdbc/cbioportal
-```
+:warning: The fields `db.host` and `db.portal_db_name` and `db.use_ssl` and `db.connection_string` are deprecated. It is required to configure the database connection using
```

Comment:
Good Fix. 

Before we completely remove the legacy properties we need to remove them from the cbioportal-core repo

https://github.com/cBioPortal/cbioportal-core/blob/1fe20d936017cdccd4d5da657bef758c56e3b711/src/main/java/org/mskcc/cbio/portal/util/DatabaseProperties.java#L61